made enable to compile chainerx with MSVC (#7108)
chainer/chainer: chainerx_cc/CMakeLists.txt

include_directories(${CMAKE_CURRENT_BINARY_DIR}/optional-lite/include)

# Test

> # TODO(durswd): Remove it. It is hack for MSVC+LLVM

Reviewer: Could you add more explanation why this hack is needed?

Author: I wrote it.

include_directories(${CMAKE_CURRENT_BINARY_DIR}/optional-lite/include)

# Test

> # ChainerX is linked with MultiThreaded DLL. So I specify gtest with MultiThreaded DLL.

Reviewer: A wording suggestion. Let's avoid ambiguous pronoun ("I").

```suggestion
# ChainerX is linked with MultiThreaded DLL, including gtest.
```

Author: I fixed it
